config/vlan.py

```diff
@@ -235,6 +237,8 @@ def del_vlan_dhcp_relay_destination(db, vid, dhcp_relay_destination_ip):
     click.echo("Removed DHCP relay destination address {} from {}".format(dhcp_relay_destination_ip, vlan_name))
     try:
         click.echo("Restarting DHCP relay service...")
-        clicommon.run_command("systemctl restart dhcp_relay", display_cmd=False)
+        clicommon.run_command("systemctl stop dhcp_relay", display_cmd=False)
+        clicommon.run_command("systemctl reset-failed dhcp_relay", display_cmd=False)
+        clicommon.run_command("systemctl start dhcp_relay", display_cmd=False)
```

Comment:
why you do not need to update the test?

Reply:
Because `display_cmd=False`, so these commands are not printed to stdout, and stdout was not being checked, so this change had no impact. I have enhanced the tests in my latest commit to ensure `clicommon.run_command()` is being called the expected number of times.